### DISCUSSION TITLE
Unable to get a data connection for wireguard VPN hosted on my Raspberry Pi connected to my pfsense router

### DISCUSSION THREAD

**Discussion by yogiberra** [pivpn/pivpn#1760] [Support]
I already have a first Raspberry Pi successfully running OpenVPN on port 1xyz and connected to my pfsense - No Issue

I installed Wireguard on a second Raspberry with port 51ABC, it connects but does not passes any data data.

Here are the data dumps
[PiVPN Wireguard data dump.txt](https://github.com/pivpn/pivpn/files/12317982/PiVPN.Wireguard.data.dump.txt)

Below is the pfsense WAN fire wall rules for Wireguard on the WAN tab.  I used the same rules for Wireguard as OpenVPN except for the port number.  Note the port number Wireguard is unique.

Any assistance will be appreciated.

Thank you

![Screenshot pFSense rules for Wireguard on my RPi](https://github.com/pivpn/pivpn/assets/24483459/45744801-8c7a-4cef-b5e2-949ea3dcbea4)

**Comment by orazioedoardo** [comment]
You have censored and changed a lot of not strictly private stuff like virtual ip addresses and ports that makes difficult to check what's going on. By the way, does tcpdump report any packet when you connect?
